internal/resources/package/data_source_package_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package tanzupackage
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	testhelper "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/testing"
+)
+
+const (
+	packageDataSourceVar = "get_package"
+	clusterResource      = "tanzu-mission-control_cluster"
+	clusterResourceVar   = "tmc_cluster_test"
+
+	packageName          = "1.7.2+vmware.1-tkg.1"
+	packageNamespaceName = "tanzu-package-repo-global"
+	packageMetadataName  = "cert-manager.tanzu.vmware.com"
+)
+
+func TestAcceptanceForSecretDataSource(t *testing.T) {
+	var provider = initTestProvider(t)
+
+	// secretName := acctest.RandomWithPrefix("tf-sct-test")
+	clusterName := acctest.RandomWithPrefix("tf-cluster-data")
+	dataSourceName := fmt.Sprintf("data.%s.%s", ResourceName, packageDataSourceVar)
+	// resourceName := fmt.Sprintf("%s.%s", ResourceName, secretResourceVar)
```

Comment:
Remove commented code.

Where are the mock acceptance tests and documentation? @Shubbhang351